subsystem/cluster_test.go

```diff
@@ -1427,6 +1441,29 @@ var _ = Describe("cluster install", func() {
 				},
 			)
 		})
+
+		// Context("Report custom features on update", func() {
+		// 	It("ARM architecture usage", func() {
+		// 		registerClusterReply, err := userBMClient.Installer.RegisterCluster(ctx, &installer.RegisterClusterParams{
+		// 			NewClusterParams: &models.ClusterCreateParams{
+		// 				BaseDNSDomain:         "example.com",
+		// 				ClusterNetworks:       []*models.ClusterNetwork{{Cidr: models.Subnet(clusterCIDR), HostPrefix: 23}},
+		// 				ServiceNetworks:       []*models.ServiceNetwork{{Cidr: models.Subnet(serviceCIDR)}},
+		// 				Name:                  swag.String("sno-cluster"),
+		// 				OpenshiftVersion:      swag.String(snoVersion),
+		// 				PullSecret:            swag.String(pullSecret),
+		// 				SSHPublicKey:          sshPublicKey,
+		// 				VipDhcpAllocation:     swag.Bool(false),
+		// 				NetworkType:           swag.String("OVNKubernetes"),
+		// 				CPUArchitecture:       "arm64",
+		// 				UserManagedNetworking: swag.Bool(true),
+		// 			},
+		// 		})
+		// 		Expect(err).ToNot(HaveOccurred())
+		// 		cluster = registerClusterReply.Payload
+
+		// 	})
+		// })
```

Comment:
why this code is commented out?

Reply:
Missed it - should be deleted. pushing an update